data/J-CODE07.json

```
@@ -5,7 +5,8 @@
   "tags": ["Open Source", "DevOps"],
   "Socials": [
     { "icon": "FaTwitter", "url": "https://twitter.com/jagan_1607" },
-    { "icon": "FaGithub", "url": "https://github.com/J-CODE07" }
+    { "icon": "FaGithub", "url": "https://github.com/J-CODE07" },
+    { "icon": "FaHashnode", "url": "https://j-code.hashnode.dev/" }
```

You probably be using an invalid icon here. It should be `SiHashnode`.

```suggestion
    { "icon": "SiHashnode", "url": "https://j-code.hashnode.dev/" }
```